src/teraranger_evo.cpp

```diff
@@ -162,8 +162,14 @@ bool TerarangerHubEvo::processAck(uint8_t* ack_buffer, const uint8_t* cmd)
         {
           return true;
         }
+        else if (ack_buffer[2] == NACK_VALUE)
+        {
+          ROS_ERROR("[%s] Command was not acknowledged", ros::this_node::getName().c_str());
+          return false;
+        }
         else
         {
+          ROS_ERROR("[%s] Invalid acknowledgment value", ros::this_node::getName().c_str());
```

Comment:
:+1:

Reply:
If everything is good for you, I will merge this PR.

Reply:
Yessir, go ahead.